libraries/AP_Mount/AP_Mount_Siyi.h

```diff
@@ -238,6 +237,8 @@ class AP_Mount_Siyi : public AP_Mount_Backend
     bool _last_record_video;                        // last record_video state sent to gimbal
 
     // absolute zoom control.  only used for A8 that does not support abs zoom control
+    ZoomType _zoom_type;                            // current zoom type
```

Comment:
This would look better in a little structure now - lots of related state :-)

Reply:
> This would look better in a little structure now - lots of related state :-)

Done